Rakefile

```diff
@@ -33,6 +33,15 @@ task release_all: [:webpack] do
   end
 end
 
+desc "Makes sure all official locales are complete and clean."
+task :check_locale_completeness do
+  DECIDIM_GEMS.each do |gem_name|
+    Dir.chdir("#{File.dirname(__FILE__)}/decidim-#{gem_name}") do
+      system({ "ENFORCED_LOCALES" => "en,ca,es" }, "rspec spec/i18n_spec.rb")
```

Comment:
Oh, sorry about that, I didn't realise it was two different things! About the official maintained locales: I'm not sure if we should even include Catalan & Spanish, isn't it a bit arbitrary?